testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java

```diff
@@ -238,12 +249,15 @@ public void onAcquireTokenClicked(final AcquireTokenFragment.RequestOptions requ
     }
 
     public void onRemoveUserClicked(final String username) {
-        mApplication.getAccounts(new PublicClientApplication.AccountsLoadedCallback() {
-            @Override
-            public void onAccountsLoaded(List<IAccount> accountsToRemove) {
-                for (final IAccount accountToRemove : accountsToRemove) {
-                    if (TextUtils.isEmpty(username) || accountToRemove.getUsername().equalsIgnoreCase(username.trim())) {
-                        mApplication.removeAccount(
+        if (mApplication instanceof IMultipleAccountPublicClientApplication){
+            final IMultipleAccountPublicClientApplication application = (IMultipleAccountPublicClientApplication)(mApplication);
```

Comment:
This feels clunky

Reply:
I refactored the test app (basically make a wrapper around MSAL. Will send out in a separate PR. Hopefully it helps.